donkeycar/utils.py

```diff
@@ -480,6 +480,12 @@ def get_model_by_type(model_type, cfg):
     elif model_type == "fastai":
         from donkeycar.parts.fastai import FastAiPilot
         kl = FastAiPilot()
+    elif model_type == "transfer":
```

Comment:
this looks as if it does not belong to this PR.
Make a new one?

Reply:
It does, because its a useful illustration on how to add new models to the new training pipeline.